### PR TITLE
Fix to allow for tokens

### PR DESCRIPTION
**Pull request by footballencarta**
As using IAM roles, rather than supplying keys directly, you also supply a token along with a key and secret.

If the token isn't supplied, the connection to DynamoDb is unsuccessful. This fix is needed to allow them to connect to DynamoDb.

---
This change is [Reviewable](https://reviewable.io/reviews/baopham/laravel-dynamodb/70)

### CODE REVIEW

File: src/DynamoDbServiceProvider.php [hunk 1]
```diff
@@ -40,11 +40,18 @@ protected function bindForApp($marshalerOptions = [])
             $config = [
                 'credentials' => [
                     'key' => config('services.dynamodb.key'),
-                    'secret' => config('services.dynamodb.secret'),
+                    'secret' => config('services.dynamodb.secret')
```
Comment (baopham) [inline src/DynamoDbServiceProvider.php]: what happen if we just put `token` here:

```
'token' => config('services.dynamodb.token')
```

Comment (footballencarta) [inline src/DynamoDbServiceProvider.php]: It does still work, however I don't believe a parameter we know may or may not be set should be passed through.

I can make the change if you wish, as it doesn't affect functionality.

Comment (baopham) [inline src/DynamoDbServiceProvider.php]: I think it's ok. Btw, could you also update README?

Comment (footballencarta) [inline src/DynamoDbServiceProvider.php]: Updated 😃  I've tried to be explicit with the fact the parameter is used for roles.

Comment (baopham) [inline src/DynamoDbServiceProvider.php]: Awesome 😄 . Feel free to update the code as discussed:
```php
'token' => config('services.dynamodb.token')
```

Comment (footballencarta) [inline src/DynamoDbServiceProvider.php]: Sorry - misunderstood your comment - code updated!

File: src/DynamoDbServiceProvider.php [hunk 2]
```diff
@@ -41,10 +41,12 @@ protected function bindForApp($marshalerOptions = [])
                 'credentials' => [
                     'key' => config('services.dynamodb.key'),
                     'secret' => config('services.dynamodb.secret'),
+                    'token' => config('services.dynamodb.token')
```
Comment (baopham) [inline src/DynamoDbServiceProvider.php]: Trailing `,` please :)

Comment (footballencarta) [inline src/DynamoDbServiceProvider.php]: Done

**Pull request merged**